src/main/java/io/nats/client/JetStreamManagement.java

```diff
@@ -121,6 +121,28 @@ public interface JetStreamManagement {
      */
     ConsumerInfo addOrUpdateConsumer(String streamName, ConsumerConfiguration config) throws IOException, JetStreamApiException;
```

Comment:
The ADR mentions changing `addConsumer` to `createOrUpdateConsumer` https://github.com/nats-io/nats-architecture-and-design/pull/275

Should we deprecate `addOrUpdateConsumer` and move to using `createOrUpdateConsumer` as well? (would be the exact same implementation, just the name changed)

Reply:
It's already commonly used, so I don't want to change it.

Reply:
Think so as well, just wanted to mention it 🙂